packages/collector/test/tracing/cloud/aws-sdk/v3/kinesis/util.js

```diff
@@ -4,54 +4,63 @@
 
 'use strict';
 
-const { KinesisClient, DescribeStreamCommand, DeleteStreamCommand } = require('@aws-sdk/client-kinesis');
+const semver = require('semver');
 
-const kinesis = new KinesisClient({ region: 'us-east-2' });
-const interval = 1000;
-const MAX_WAIT_TIME = 10000;
+const minNodeJsVersion = '14.0.0';
 
-exports.checkStreamExistence = function checkStreamExistence(streamName, expectsToExist = true) {
-  let timer = 0;
-  return new Promise(resolve => {
-    const intervalId = setInterval(async () => {
-      timer += interval;
-      if (timer >= MAX_WAIT_TIME) {
-        resolve(`Timeout after ${timer} ms`);
-        clearInterval(intervalId);
-      }
-      const command = new DescribeStreamCommand({ StreamName: streamName });
+if (semver.lt(process.versions.node, minNodeJsVersion)) {
```

Comment:
suggestion: require the util after the node version check in the test. IMO it is hard to maintain two version checks for one test.

Reply:
Updated the code as per the comment.